src/debug.c

```diff
@@ -872,13 +872,15 @@ NULL
         nanosleep(&tv, NULL);
         addReply(c,shared.ok);
     } else if (!strcasecmp(c->argv[1]->ptr,"set-active-expire") &&
-               c->argc == 3)
-    {
+               c->argc == 3) {
         server.active_expire_enabled = atoi(c->argv[2]->ptr);
         addReply(c,shared.ok);
+    }else if (!strcasecmp(c->argv[1]->ptr,"set-allow-access-expired") &&
```

Comment:
```suggestion
    } else if (!strcasecmp(c->argv[1]->ptr,"set-allow-access-expired") &&
```

please add the document for this command in the top of this method.